Sources_v3/StreamChatUI/ChatChannel/MessageComposer/MessageComposerViewController.swift

```diff
@@ -384,15 +385,16 @@ open class MessageComposerViewController<ExtraData: ExtraDataTypes>: ViewControl
         composerView.invalidateIntrinsicContentSize()
     }
     
-    // MARK: UITextView
+    // MARK: Suggestions
 
-    @objc func promptSuggestionIfNeeded(for text: String) {
-        if text.trimmingCharacters(in: .whitespacesAndNewlines).contains("@") {
-            if let index = (text.range(of: "@")?.upperBound) {
+    private func promptSuggestionIfNeeded(for text: String) {
```

Comment:
the rule is - unless there's a strong reason for not having something `open` it should be open. Tell me why `promptSuggestionIfNeeded(for text: String)` should not be `open` please :)

Reply:
I guess you are right, maybe I didn't think too much high-level and started to complicate stuff, I thought that this function should be called only from the `textViewDidChange` so it makes kinda sense for the function to be private. Changed now.